.github/workflows/deploy-storybook.yml

```diff
@@ -34,7 +37,13 @@ jobs:
         if: steps.yarn-cache.outputs.cache-hit != 'true'
         run: yarn install
 
-      - name: Deploy Storybook to Github page
-        run: yarn storybook:deploy --ci
-        env:
-          GH_TOKEN: trussworks:${{ secrets.GITHUB_TOKEN }}
+      - name: Build Storybook
+        run: yarn build-storybook --modern
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          clean: true
+          clean-exclude: |
+            CNAME
```

Comment:
I read in the commit that this is _if_ it exists, but where is this meant to be deployed today? Do we have a URL for this now? Should we make a subdomain for the `truss.dev` domain?

Reply:
I put it in since `clean: true` would remove it, in case we decided to have a custom domain. But maybe it's better to leave it out until we make that choice? We don't have a domain for it now but I'm not opposed to a custom one. Likely there are links in the repo we'd have to adjust.